src/backend/router.ts

```diff
@@ -28,22 +33,35 @@ export default (texasClient: TexasClient, router: Router) => {
     });
 
     // APP
+    if (!isProd) {
+        vite = await createViteServer({
+            root: path.join(process.cwd(), 'src/frontend'),
+            server: { middlewareMode: true },
+            appType: 'custom',
+        });
+        router.use(vite.middlewares);
+    }
+
     router.get(
         ['/', '/fagsystem/*splat'],
         ensureAuthenticated(texasClient, false),
-        (req: Request, res: Response): void => {
+        async (req: Request, res: Response): Promise<void> => {
             prometheusTellere.appLoad.inc();
             const csrfToken = genererCsrfToken(req.session);
+            const url = req.originalUrl;
             try {
                 let htmlInnhold = fs.readFileSync(
-                    `${path.join(process.cwd(), buildPath)}/index.html`,
-                    'utf8'
-                );
-                htmlInnhold = htmlInnhold.replace(
-                    'content="{{ csrf_token() }}"',
-                    `content="${csrfToken}"`
+                    path.join(process.cwd(), isProd ? buildPath : 'src/frontend', 'index.html'),
```

Comment:
Ville foretrukket om man heller hadde hatt to forskjellige funksjoner som returnerer hele HTML-en basert på hvilke miljø man er i 🤔 Så kan man kun gjøre CSRF logikk her